common/src/main/resources/startup.properties

```diff
@@ -332,3 +333,6 @@ it.workflow.execution.listeners=org.apache.falcon.catalog.CatalogPartitionHandle
 #*.falcon.graphite.port=2003
 #*.falcon.graphite.frequency=1
 #*.falcon.graphite.prefix=falcon
+
+# Property to remove postProcessing
+*.falcon.postprocessing.enable=true
```

Comment:
Well, runtime properties are system level settings and will affect all users (only admins make this change). So, one user can't play around with it.

I understand the behavior change that users will notice, but, that is expected and holds for all other properties in runtime.properties too. For example, *.feed.retry.attempts, users will see a change in number of retries based on the value here.

I prefer it being in runtime.properties for the simple reason that this is not something required during startup and an admin can modify it without having to restart the server. Having said that, I don't want to be hung upon where it should go.